### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by mendentm**
I added in a lighter green color and enlarged the font sizes for the nav slightly, and the paragraph under H1. I also wrote a short and weird bio. Let me know what you think!

### CODE REVIEW

**Comment by timmyd61** [comment]
I like it. I'll let Dami weigh in before merging.

**Review by milya1 — CHANGES_REQUESTED** [review]
Love the bio! Do you think we should include our photos there as well? :)

**Review by milya1 — APPROVED** [review]

**Pull request merged**
